fjord/__init__.py

```diff
@@ -1,7 +1,16 @@
+# First we import absolute imports from the future, so that our celery.py
+# module will not clash with the library
+from __future__ import absolute_import
+
 import os
 import site
 import sys
 
+from fjord.manage_utils import monkeypatch
+
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
```

Comment:
Note, that doesn't work in the vendor setup. I had to create a new virtualenv with requirements installed in it to test this path. I guess it should work in vagrant too.